partner/WHMCS/modules/servers/jetpack/jetpack.php

```diff
@@ -84,12 +84,13 @@ function jetpack_CreateAccount(array $params)
         );
 
         $response = make_api_request($provisioning_url, $access_token, $request_data);
+        if ($response->success && $response->success == true) {
+            if ($response->next_url) {
+                save_provisioning_details($response->next_url, $params);
+            } else if(!$response->next_url && $response->auth_required)  {
+                save_provisioning_details($response->next_url, $params, true);
+            }
 
-        if (isset($response->next_url)) {
-            save_next_url($response->next_url, $params);
-        }
-
-        if ($response->success == true) {
             return 'success';
         }
```

Comment:
What if `$response->success` is false?

Reply:
@ebinnion good question and catch. I've updated the make_api_request to always return the response in the event of a 4xx error and this will also be returned for a 2xx. If this is false or $response->success is anything other than true this will now be returned as well which will add a log entry to the Activity Log.